evalml/pipelines/regression_pipeline.py

```diff
@@ -26,9 +25,9 @@ def fit(self, X, y):
         """
         X = _convert_to_woodwork_structure(X)
         y = _convert_to_woodwork_structure(y)
+        if "numeric" not in y.semantic_tags:
```

Comment:
Is there an `is_numeric` util method in woodwork which we can use? Or is this the recommended way to do this check? @gsheni

Reply:
Yes, there is (thought it wasn't meant to be user facing).
https://github.com/alteryx/woodwork/blob/main/woodwork/datacolumn.py#L318